static/js/components/CourseList.js

```diff
@@ -33,17 +40,25 @@ class CourseList extends React.Component {
         </ul>;
       });
 
-      return <div key={program.id} className="course-list">
-        <ul className="course-list-header">
-          <li className="course-title">
-            {program.title}
-          </li>
-          <li className="course-status "/>
-          <li className="course-progress">
-            Progress
-          </li>
-        </ul>
-        {table}
+      return <div key={program.id + "_1"}>
```

Comment:
We only need the key attribute at the root element. Can you remove the `key={program.id}` below and change this to `key={program.id}`?